packages/docs/developer-resources/contractkit/usage.md

```diff
@@ -20,9 +20,20 @@ await kit.setFeeCurrency(CeloContract.StableToken)
 This method from the `kit` will return the cGLD, locked cGLD, cUSD and total balance of the address 
 
 ```ts
-await kit.getTotalBalance(myAddress)
+let totalBalance = await kit.getTotalBalance(myAddress)
 ```
 
+## Deploy a contract
+
+Deploying a contract with the default account already set. Simply send a transaction with no `to:` field. See more about [sending custom transactions](https://docs.celo.org/developer-guide/overview/introduction/contractkit/contracts-wrappers-registry#sending-custom-transactions)
+
+```ts
+let bytecode = '0x608060405234...' // compiled Solidity deployment bytecode
+
+let tx = await kit.sendTransaction({
```

Comment:
maybe add a link to blockscout and/or how to check that the deploy was successful?